components/postmark/postmark.app.mjs

```diff
@@ -1,11 +1,208 @@
+import axios from "axios";
+import { axios as axiosPipedream } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "postmark",
-  propDefinitions: {},
+  propDefinitions: {
+    from_email: {
```

Comment:
Typically we use `camelCase` for prop names even if the API accepts only `snake_case`. It would be _converted_ when constructing the `params` or `data` object. For example:

```js
propDefinitions: {
  fromEmail: {
    ...
  },
  ...
},
```

```js
component.mjs

props: {
  fromEmail: {
    propDefinition: [
      postmark,
      "fromEmail",
    ],
  },
}.
...
async run({ $ }) {
  const params = {
    from_email: this.fromEmail,
    ...
  };
  ...
}

```

Reply:
Adjusting the prop names. These were props from the existing Postmark action that I moved to the app's propDefinitions, but did not pay attention to the naming - in fact, this API expects `PascalCase` object properties, so they were already being converted. Nonetheless, you provided perfect examples 👍